### PR TITLE
Add Pinecone configuration

### PR DESCRIPTION
**Pull request by illegalnumbers**

### CODE REVIEW

**Review by nlu90 — APPROVED** [review]

File: connectors/pinecone-sink/pinecone-sink.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+repository: https://github.com/streamnative/pulsar-io-pinecone
+icon: "/images/connectors/pinecone-logo.png"
+sn_available: true
```
Comment (shibd) [inline connectors/pinecone-sink/pinecone-sink.yaml:3]: Before the sn cloud, this value should be false.

Comment (illegalnumbers) [reply connectors/pinecone-sink/pinecone-sink.yaml:3]: Can you clarify what you mean by that? I do not understand

Comment (shibd) [reply connectors/pinecone-sink/pinecone-sink.yaml:3]: This value serves as a label on the [pulsar-hub](https://docs.streamnative.io/hub). If it's true, it means it's built-in on our SN Cloud.


pulsar-hub:
image

sn-cloud:
image

Comment (illegalnumbers) [reply connectors/pinecone-sink/pinecone-sink.yaml:3]: I will be adding it soon

**Pull request merged**